main.py

```
@@ -376,6 +377,104 @@ async def metrics():
+                
+            except WebSocketDisconnect:
+                break
+            except Exception as e:
+                logger.exception("WebSocket message handling error", user_id=user_id, path=path)
+                try:
+                    await websocket.send_json({
+                        "type": "error",
+                        "message": str(e)
+                    })
+                except Exception:
+                    pass  # Connection might be closed
```

Ruff (0.12.2)

`try`-`except`-`pass` detected, consider logging the exception

(S110)